mentat/code_context.py

```diff
@@ -116,41 +117,72 @@ async def get_code_message(
         # we just refresh the cache every time get_code_message is called
         self.diff_context.refresh()
         if self.diff_context.diff_files():
-            code_message += [
-                "Diff References:",
-                f' "-" = {self.diff_context.name}',
-                ' "+" = Active Changes',
-                "",
-            ]
+            code_message += [f"Diff References: {self.diff_context.name}\n"]
 
-        code_message += ["Code Files:\n"]
+        code_message += ["Code Files:\n\n"]
 
-        # Calculate user included features token size
-        include_features = [feature for file_features in self.include_files.values() for feature in file_features]
+        await self.daemon.update()
+        context_builder = self.daemon.get_context("", max_tokens=0)
```

Comment:
@PCSwingle - I had an idea about what you brought up on slack, re: included vs auto context. Check out this `ContextBuilder` class from Ragdaemon: https://github.com/AbanteAI/ragdaemon/blob/main/ragdaemon/context.py

You load paths, refs or diffs in, can add/remove with line numbers, then it has a `.render()` method that produces our `code_message`. Here, I load it up with `included_files`, then on line 167 I'll add auto-token files.

In a previous version of this commit, I made CodeContext.include_files a ContextBuilder straightaway, and maintained it like that. It has .add .remove methods to go with our commands. We could keep one, persistent ContextBuilder throughout the session, starting with your included_files (like I did before), and adding your auto-stuff to it, so you do just have one context. 

THis means your auto-context would be persistent, different from now, but I think that's a good thing. The *content* of the files your seeing should be up to date, but it doesn't need to re-select context after the first message.